### PR TITLE
es 7.6.0 support

### PR DESCRIPTION
**Pull request by clutroth**
🧐Enhancement (ES) ES 7.6.0 support

### CODE REVIEW

File: es74x-cloud/gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-esVersion=7.5.1
```
Comment (clutroth) [inline es74x-cloud/gradle.properties]: @coutoPL

Comment (coutoPL) [reply es74x-cloud/gradle.properties]: it's ok

**Comment by sscarduzio** [comment]
Hey guys, yesterday night on my laptop I had 100% integration test pass forcing es 7.6.0 dependency on the pre-existing es74x subproject. 

So why did we need to create es76x? I'm sure maybe there are reasons I didn't see?

The philosophy behind this question being the following: 
(I'm not sure if we ever talked about this, so let's make sure we are on the same page)

It's imperative to keep the code base at the bare minimum. I think this is valid for ANY code base. More code ALWAYS results (or will result) in more problems. 

As developers, "more code" is our enemy. As developers in security, "more code" is the antichrist.

Therefore, unless there's a significant API change that breaks the current esXXx subproject, let's keep reusing it for the new ES versions. 
I'd even be ready to use some reflection hacks to keep us from creating another subproject.

**Comment by coutoPL** [comment]
OMG, @sscarduzio please take a look at files changed.

**Comment by sscarduzio** [comment]
Shit I have to go to bed earlier. Sorry lol.

**Pull request merged**